OmarJabari/task.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+
+class Task:
+    id_counter = 1
+    def __generate_id(self):
```

Comment:
There is nothing to update here, but there's something you'll need to consider. This is okay, but if you implemented the Data Persistence Bonus part, it will be invalid.